frontend/src/GroupProfilePage/TopGenres/TopGenres.js

```diff
@@ -14,23 +36,23 @@ export const TopGenres = (props) => {
             <div className="genre-bubbles">
                 <div className="top-genre-1">
                     <div className="bubble"></div>
-                    <h2><strong>{top_genres[0]}</strong></h2>
+                    <h2><strong>{top_genres[0].genre}</strong></h2>
                 </div>
                 <div className="top-genre-2">
                     <div className="bubble"></div>
-                    <h2><strong>{top_genres[1]}</strong></h2>
+                    <h2><strong>{top_genres[1].genre}</strong></h2>
                 </div>
                 <div className="top-genre-3">
                     <div className="bubble"></div>
-                    <h2><strong>{top_genres[2]}</strong></h2>
+                    <h2><strong>{top_genres[2].genre}</strong></h2>
                 </div>
                 <div className="top-genre-4">
                     <div className="bubble"></div>
-                    <h2><strong>{top_genres[3]}</strong></h2>
+                    <h2><strong>{top_genres[3].genre}</strong></h2>
                 </div>
                 <div className="top-genre-5">
                     <div className="bubble"></div>
-                    <h2><strong>{top_genres[4]}</strong></h2>
+                    <h2><strong>{top_genres[4].genre}</strong></h2>
```

Comment:
I would be careful with accessing the attribute of an array index because if the index is out of the range of the array, the application might throw an error. A possible fix is to check if the array at the index isn't undefined, then render the element.

Reply:
ah yes I will fix that